content.en/discv5/methodology.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "discv5 reports methodology"
+plotly: true
+slug: methodology
+---
+
+# discv5 reports methodology
+
+## DHT Crawling
```

Comment:
This is a bit redundant to https://probelab.io/tools/nebula/#crawler but I think it's still relevant because of the special handling of things in discv5.

Perhaps just link to that other section?

Reply:
Good point!